layouts/partials/contentNavigation.html

```diff
@@ -0,0 +1,87 @@
+<aside id="aside" class="o-aside o-aside__bottom-sheet">
```

Comment:
Since the same HTML is used for the aside and the bottom-sheet, shouldn't we just take `o-aside` and remove the "bottom-sheet" prefix?

`o-aside__bottom-sheet` > `o-aside`
`o-aside__bottom-sheet-header` > `o-aside__header`
`o-aside` > `o-aside__drag`
and so on.

The only conflict I see it the news sidebar, but maybe we can consistently update the classes there.